components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -578,10 +578,13 @@
 }
 #current-phenotype-selection .summary-item label {
   width: auto;
-  margin-right: 1em;
+  margin-right: 0.3em;
+}
+#current-phenotype-selection .summary-item .tools .xHelpButton {
+  margin : 0.6em;
 }
 #current-phenotype-selection .summary-item .tools {
-  color: $theme.textPrimaryColor;
+  color: #636B75;
```

Comment:
Hardcoded values are bad, they don't play nice with different color themes. Please find a color from the theme to use instead.